core-feature-pack/src/main/resources/content/bin/common.ps1

```diff
@@ -167,19 +177,22 @@ Param(
   $PROG_ARGS += "-Djboss.server.config.dir=$global:JBOSS_CONFIG_DIR"
 
   if ($GC_LOG -eq $true){
-    if ($PROG_ARGS -notcontains "-verbose:gc"){
+    $dir = New-Item $JBOSS_LOG_DIR -type directory -ErrorAction SilentlyContinue
+    if ($PROG_ARGS -notmatch "-Xlog:?gc"){
         Rotate-GC-Logs
-		if (-not(Test-Path $JBOSS_LOG_DIR)) {
-			$dir = New-Item $JBOSS_LOG_DIR -type directory -ErrorAction SilentlyContinue
-		}
-        $PROG_ARGS += "-verbose:gc"
-        $PROG_ARGS += "-XX:+PrintGCDetails"
-        $PROG_ARGS += "-XX:+PrintGCDateStamps"
-        $PROG_ARGS += "-XX:+UseGCLogFileRotation"
-        $PROG_ARGS += "-XX:NumberOfGCLogFiles=5"
-        $PROG_ARGS += "-XX:GCLogFileSize=3M"
-        $PROG_ARGS += "-XX:-TraceClassUnloading"
-        $PROG_ARGS += "-Xloggc:$JBOSS_LOG_DIR\gc.log"
+
+        if ($MODULAR_JDK -eq $true)
+        {
+            $PROG_ARGS += "-Xlog:gc*:file=`\`"$JBOSS_LOG_DIR\gc.log`\`":time,uptimemillis:filecount=5,filesize=3M"
+        } else {
+            $PROG_ARGS += "-XX:+PrintGCDetails"
```

Comment:
That shouldn't be needed I don't think, but it I guess for consistency we should probably add it back.

Reply:
I check the also ps1 and sh scritps. `-verbose:gc` was removed also from sh and bat scripts in JDK8 usage. This PR shouldn't do any changes in JDK8 behavior.

I did deeper JDK8 param check.

----

Sh script:
Originally, there was `PREPEND_JAVA_OPTS="$PREPEND_JAVA_OPTS -verbose:gc -Xloggc:\"$JBOSS_LOG_DIR/gc.log\"`

This PR chage this to `TMP_PARAM="-Xloggc:$JBOSS_LOG_DIR/gc.log`

@rstancel What is the reason for the removing of quotation for `-Xloggc` param?

Reply:
@marekkopecky the whole logic was rewritten so also quotes were updated to match this logic. Is there any issue with it?

Reply:
@rstancel I don't see any issue, but AFAIK, WF users can use our scripts in creative way, so we shouldn't change anything, that works correctly.

> the whole logic was rewritten so also quotes were updated to match this logic.

@rstancel Can you please share more details? If I add escaped quotes to `TMP_PARAM` and `eval` to test line 287, scripts seems to work correctly ...

Reply:
@marekkopecky from my point of view there were two places where we called exactly same larger sets of arguments. With bringing JDK_MODULE logic it was increased to two such sets. And with consideration the fact that these are not simple calls it is quite possibly that someone (we, WF users...) will do the mistake and will not copy these parameters correctly in both places. So my point here was to make this logic a little simpler and more robust to avoid possible issues.

I see your point not to change things which are working correctly but it is also our responsibility to bring to code the stability and innovation.